.pre-commit-hooks.yaml

```diff
@@ -85,6 +85,22 @@
   language: script
   files: \.go$
 ######################
+# OpenAPI related hooks
+- id: openapi-lint
+  name: "openapi-lint: Lint OpenAPI specifications"
+  description: Runs Spectral linting against OpenAPI specification files
+  args: [
+    lint,
+    spec/**/*.yaml,
```

Comment:
@turo/devops, if anyone has any suggestions, I'd really like the directory containing specs to not be specified here but by the repo that uses the pre-commit hook.  I haven't yet figured out a way to do that.